Libraries/LibWeb/WebAssembly/WebAssembly.cpp

```diff
@@ -237,12 +248,13 @@ JS::ThrowCompletionOr<NonnullOwnPtr<Wasm::ModuleInstance>> instantiate_module(JS
                     // https://webassembly.github.io/spec/js-api/#read-the-imports step 5.1
                     if (import_.is_number() || import_.is_bigint()) {
                         if (import_.is_number() && type.type().kind() == Wasm::ValueType::I64) {
-                            // FIXME: Throw a LinkError instead.
-                            return vm.throw_completion<JS::TypeError>("LinkError: Import resolution attempted to cast a Number to a BigInteger"sv);
+                            return vm.throw_completion<LinkError>("LinkError: Import resolution attempted to cast a Number to a BigInteger"_string);
```

Comment:
`LinkError: ` prefix seems to be not necessary here (and in other places)?

Reply:
This reminds me to switch LinkError to use StringViews instead of Strings later on

Reply:
```
/Users/pavel/Develop/ladybird/Libraries/LibJS/Runtime/VM.h:208:30: error: no matching member function for call to 'create'
  208 |         auto completion = T::create(realm, forward<Args>(args)...);
      |                           ~~~^~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibWeb/WebAssembly/WebAssembly.cpp:195:35: note: in instantiation of function template specialization 'JS::VM::throw_completion<Web::WebAssembly::LinkError, const char (&)[53]>' requested here
  195 |                         return vm.throw_completion<LinkError>("LinkError: Imported value is not a callable function");
      |                                   ^
/Users/pavel/Develop/ladybird/Libraries/LibWeb/WebAssembly/Error.h:26:31: note: candidate function not viable: no known conversion from 'const char[53]' to 'String' for 2nd argument
   26 |     static GC::Ref<LinkError> create(JS::Realm&, String message);
      |                               ^                  ~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibWeb/WebAssembly/Error.h:25:31: note: candidate function not viable: requires 1 argument, but 2 were provided
   25 |     static GC::Ref<LinkError> create(JS::Realm&);
      |                               ^      ~~~~~~~~~~
1 error generated.
```

Reply:
compiles for me on the freshest `master` (a74954e1); it seems that you’ve got a different issue on ci–prefixing `Error` might do the job there.

Reply:
There's no LinkError on CI.... Do you mean you rebased?

Reply:
I switched to StringViews